src/collider/README.md

```diff
@@ -38,3 +37,49 @@ A websocket-based signaling server in Go.
 
     go test collider
 
+## Deployment
+
+Change [roomSrv](https://github.com/webrtc/apprtc/blob/master/src/collider/collidermain/main.go#L16) to your AppRTC server instance e.g.
+
+```go
+var roomSrv = flag.String("room-server", "https://your.apprtc.server", "The origin of the room server")
+```
+
+Then repeat step 6 in the Building section and then start it on your server by following the Running instructions.
+
+### Certificates
+If you are deploying this in production, you should use certificates so that you can use secure websockets. Place the `cert.pem` and `key.pem` files in `/cert/`. E.g. `/cert/cert.pem` and `/cert/key.pem`
+
+### Auto restart
+1\. Add a `/collider/start.sh` file:
+
+```bash
+/usr/local/bin/node /node/send_restart_alert.js $(hostname) 2>>/collider/collider.log
+/collider/collidermain 2>> /collider/collider.log
+```
+
+2\. Make it executable by running `chmod 744 start.sh`.
+
+3\. Add the following line to `/etc/inittab` to allow automatic restart of the Collider process:
+```bash
+coll:2:respawn:/collider/start.sh
```

Comment:
This assumes there's a user 'coll' on the system. I don't see any instructions mentioning setting that up.